nion/swift/ExportDialog.py

```diff
@@ -261,15 +261,29 @@ class UnitType(enum.Enum):
 
 
 class ExportSizeModel(Observable.Observable):
-    def __init__(self, display_item: DisplayItem.DisplayItem) -> None:
+    unit_to_str_map = {
```

Comment:
I know this same thing is done in other parts of the code, but I think we should use better naming for these variables - `unit_to_unit_identifier_map` and `unit_identifier_to_unit_map` are verbose, but explain better what they are. `str` is too general and is confusing with the titles used in `ExportSVGHandler`.